dbsync-central/application.prod.properties

```diff
@@ -0,0 +1,182 @@
+#                                                                                                                      #
+# This file provides a template for core configuration properties for the DB sync receiver application, it contains    #
+# in-line documentation for each property. For the spring core properties, please refer to the spring boot docs for    #
+# more details.                                                                                                        #
+#                                                                                                                      #
+# NOTE: For the properties that have preset values, it's recommended not to change them unless you have to, some of    #
+# them have placeholders of the form ${my.other.prop.key}, it means all you need to do is set values for the property  #
+# key in this file named my.other.prop.key                                                                             #
+#                                                                                                                      #
+########################################################################################################################
+
+
+# *********************** DB sync receiver properties ******************************************************************
+#
+#eip.home=${file.separator}user${file.separator}local${file.separator}dbsync
+eip.home=${home_dir}
+
+# The interval in milliseconds between polls of the retry queue by the retry route
+inbound.retry.interval=1800000
+
+# The interval in milliseconds before the first poll of the retry queue by the retry route when the application starts.
+inbound.retry.initial.delay=120000
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Logging Configuration ************************************************************************
+#
+# Uncomment this to use console logging otherwise defaults to the logback.xml file located on the classpath which is
+# configured to log to ${eip.home}/logs/openmrs-eip.log
+logging.config=file:logback-spring.xml
+#logging.config=classpath:logback-console.xml
+
+# Logging level for the application loggers (all built-in routes and classes)
+openmrs.eip.log.level=INFO
+
+# Logging level for ONLY application class loggers, feel free to add more logger configurations
+logging.level.org.openmrs.eip=${openmrs.eip.log.level}
+logging.level.receiver-main=${openmrs.eip.log.level}
+logging.level.inbound-db-sync=${openmrs.eip.log.level}
+logging.level.inbound-logger=${openmrs.eip.log.level}
+logging.level.inbound-error-handler=${openmrs.eip.log.level}
+logging.level.receiver-retry=${openmrs.eip.log.level}
+logging.level.inbound-complex-obs-sync=${openmrs.eip.log.level}
+logging.level.message-processor=${openmrs.eip.log.level}
```

Comment:
You're missing **logging.level.receiver-request-processor**

Reply:
Okay